### ISSUE TITLE
Linking to tags by `id` attributes

### ISSUE THREAD

**Issue by max-heller** [max-heller/mdbook-pandoc#39]
```markdown
<a id="test">some text here</a>
[test link](test)
```

is converted to

```latex
some text here\href{test}{test link}
```

but there should be a link target defined with the tag's `id` ("test").

Pandoc deliberately does not try to parse raw HTML, so this needs either a full-fledged HTML parsing solution (#44) or a hacky, best-effort solution

**Comment by max-heller** [comment]
Likely part of #44

**Comment by max-heller** [comment]
This should be achievable using a similar approach to #124

**Comment by max-heller** [comment]
With support for linking to `<a>` and `<span>` elements by `id` implemented in #128 and #130, most of the instances I've seen in mdbooks are now functional. If necessary, it should be possible to support other elements as well, but it's less clear in those cases what the attributes should be attached to (`<a>` and `<span>` are mapped to [Pandoc spans](https://pandoc.org/MANUAL.html#extension-bracketed_spans)).
